### ISSUE TITLE
Problem with Sharp version on Apple M1 chips

### ISSUE THREAD

**Issue by ozgrozer** [ikhsanalatsary/multer-sharp-s3#64]
There's a problem with Sharp version you're using on M1 Macbooks. It just cannot be installed.
Can you update Sharp to the latest version?

https://sharp.pixelplumbing.com/install#apple-m1

**Comment by rvanzon** [comment]
I forked the repo, updated Sharp and created a PR  #65 (tests seem to fail because of prior changes, details are not publicly available)

In the meantime you can use in your `package.json`:
```
"multer-sharp-s3": "git+https://github.com/rvanzon/multer-sharp-s3.git",
```

**Comment by alysonfreitas** [comment]
Can someone merge this? thank you @rvanzon

**Comment by abforman** [comment]
Latest would be best, but anything over 0.29 will have M1 support - hoping this merges soon!

**Comment by ikhsanalatsary** [comment]
Thank you for addressing this, I already published 0.2.2 to npm
